src/xcode/ENA/HealthCertificateToolkit/Tests/HealthCertificateToolkitTests/DigitalGreenCertificateAccessTests.swift

```diff
@@ -2,24 +2,44 @@
 // 🦠 Corona-Warn-App
 //
 
-
 import XCTest
 import SwiftCBOR
 @testable import HealthCertificateToolkit
 
-final class DigitalGreenCertificateAccessTests: CWATestCase {
+final class DigitalGreenCertificateAccessTests: XCTestCase {
 
-    func test_When_DecodeSucceeds_Then_CorrectCertificateIsReturned() {
+    func test_When_DecodeVaccinationCertificateSucceeds_Then_CorrectCertificateIsReturned() {
         let certificateAccess = DigitalGreenCertificateAccess()
         
-        let result = certificateAccess.extractDigitalGreenCertificate(from: testData.input)
+        let result = certificateAccess.extractDigitalGreenCertificate(from: testDataVaccinationCertificate.input)
+
+        if case .failure(let error) = result {
+            print(error)
```

Comment:
Does print work in our Testcases?

Reply:
Need to remove this if case completely. Its not needed, only for debugging